@stellar/anchor-tests/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stellar/anchor-tests",
-  "version": "0.5.11",
+  "version": "0.5.12",
```

Comment:
Can you set version to 0.6 please? 
AP uses 0.5.11 and if there's a hot fix necessary for current tests it'd be hard to fix them
(AP should eventually move to 0.6 but ideally we want to generate transactions from current build of the platform, so it could take a while)